src/clj/puppetlabs/services/legacy_routes/legacy_routes_core.clj

```diff
@@ -0,0 +1,144 @@
+(ns puppetlabs.services.legacy-routes.legacy-routes-core
+  (:require [clojure.string :as str]
+            [clojure.tools.logging :as log]
+            [compojure.core :as compojure]
+            [ring.util.codec :as ring-codec]
+            [compojure.route :as route]
+            [clojure.tools.logging :as log]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; v3 => v4 header translation support functions
+
+; TODO: Improve the naive parsing here with a real library like
+; https://github.com/ToBeReplaced/http-accept-headers
```

Comment:
Cool, thanks for finding that.  Do you think I should update this PR to use `parse-accepts-header` in liberator, or perhaps copy the function if we're planning on removing our dependency on liberator in the future?